jstests/libs/backup_utils.js

```diff
@@ -128,7 +128,9 @@ function _copyFileHelper(absoluteFilePath, sourceDbPath, destinationDirectory) {
     // Grab the file path relative to the dbpath. Maintain that relation when copying
     // to the `hiddenDbpath`.
     let relativePath = absoluteFilePath.substr(sourceDbPath.length);
-    let destination = destinationDirectory + '/' + relativePath;
+    let destination = destinationDirectory + separator + relativePath;
+    const newFileDirectory = destination.substring(0, destination.lastIndexOf(separator));
+    mkdir(newFileDirectory);
```

Comment:
My so-called "expertise" in JavaScript isn't sufficient enough to figure out where the `mkdir` function comes from :). I guess it is an analog of the `mkdir` command line utility. I'd like to know whether the function works as if the `mkdir -p` command is issued. Without the `-p` "option", it may fail.